src/widget-editor/ReadMe.md

```diff
@@ -0,0 +1,45 @@
+# Admin App custom widget editor
+
+With this AdminApp extensibility point, you can create your own cusom widget designers and override the defualt auto generated designers.
+
+In this example we have two provided two samples:
+
+* For overriding a specific field for a widget designer.
+
+* For overriding the entire view for a widget designer.
+
+
+## Overriding a custom field
+
+The process is exactly the same as it is with overriding a field for a content type with the exception that the typeName of the item is equal to widget-{WIDGET_NAME}.
+
+The process invloves registering a custom field provider class by inheriting from the FieldsProvider provider interface and implementing the method "overrideField". This method is called for each and every field when it is beeing rendered in the interface. Thus you are able to replace each field.
```

Comment:
Lets add link to the API docs as in the other readme files. For example **FieldsProvider** to be a link to: http://admin-app-extensions-docs.sitefinity.site/interfaces/fieldsprovider.html
overrideFields -> http://admin-app-extensions-docs.sitefinity.site/interfaces/fieldsprovider.html#overridefield
etc.